components/Apply.js

```diff
@@ -22,44 +22,45 @@ const Apply = () => {
             <Image src="/donate2.png" alt="Donate" width={250} height={30} />
           </div>
           <p className="text-sm mt-8">
-            By supporting students, you&apos;re becoming an active participant in
-            building their careers. You can easily cancel or upgrade your
-            contribution at any time.
+            By supporting students, you are becoming an active participant in
+            building their careers. All donations and contributions directly sponsor the attempts of talented students to 
+            secure admission in an MS/PhD program at a university of international repute. You can easily cancel or upgrade your contribution
+            at any time.
           </p>
         </div>
+        <a href="https://amu.ac.in/">
         <button className="px-2 py-2 mt-8 w-full bg-red-850 text-sm text-white text-center">
-          SSGSA results (2021-2022) announce
+          The Aligarh Muslim University
         </button>
+        </a>
+        <a href="http://ssgsa.us/uploads/PressRelease-SSGSA-2021-22.pdf">
```

Comment:
This shouldn't be done this way. You have to upload the document in this project and then the reference to be given.

Reply:
Also, for such anchor tags, give target as blank. It is good to open documents in other tabs.